docs/build/guides/dapps/working-with-contract-specs.mdx

```diff
@@ -327,16 +327,21 @@ def send_transaction() -> soroban_rpc.SendTransactionResponse:
     root_keypair = Keypair.from_secret(
         "SXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
     )
-    root_account = server.load_account("GBSBL6FBPX5UHKL4AZCPUU6PXKUBYMKRUN3L4YQ4V2CCWSE7YMN2HYPB")
+    root_account = Account(account=root_keypair.public_key, sequence=1)
     contract_id = "CA3D5KRYM6CB7OWQ6TWYRR3Z4T7GNZLKERYNZGGA5SOAOPIFY6YQGAXE"
     transaction = (
         TransactionBuilder(
             source_account=root_account,
             network_passphrase=Network.TESTNET_NETWORK_PASSPHRASE,
             base_fee=100,
         )
-        .append_invoke_contract_function_op(contract_id,"increment")
-         # mark this transaction as valid only for the next 30 seconds
+        .append_invoke_contract_function_op(
+            contract_id=contract_id,
+            function_name="increment",
+            parameters=[],
+            source=root_keypair.public_key,
```

Comment:
we can omit the `source=...` account in the operation, since it can inherit from the transaction. not sure about `parameters`, though. does the sdk require us to have the empty list now?